content-src/selectors/siteMetadataSelectors.js

```diff
@@ -8,7 +8,12 @@ function selectSiteProperties(site) {
   const metadataFavicon = site.favicons && site.favicons[0] && site.favicons[0].url;
   const favicon = site.favicon_url || metadataFavicon || site.favicon;
   const parsedUrl = site.parsedUrl || urlParse(site.url || "");
-  const label = prettyUrl(parsedUrl.hostname);
+
+  // Remove the eTLD (e.g., com, net) and the preceding period from the hostname
+  const eTLDLength = (site.eTLD || "").length;
+  const eTLDExtra = eTLDLength > 0 ? -eTLDLength - 1 : Infinity;
```

Comment:
Can you put parenthesis around the subtraction? I think it would help readability since there's already the shorthand if statement there.